### DISCUSSION TITLE
Bug: App quits / crashes when using transform tool

### DISCUSSION THREAD

**Discussion by falkthepolice** [keiwando/toonsquid#84] [Q&A]
Howdy,

First off I love the app (an actual vector animation app!) but I seem to have hit a bit of a wall - the app now crashes every time I try to use the transform tool - I’ve tried clearing the cache and restarting my iPad (Pro 12.9-inch 5th generation) but no luck. Any suggestions would be appreciated.

thanks

https://user-images.githubusercontent.com/119546893/204937508-02f5c8f2-e9cb-48a4-9462-c6105b1f95de.MOV

**Comment by keiwando** [comment]
Hi! Thank you for reporting this issue. Would it be possible for you to send me the project file to contact@toonsquid.com so I could try to reproduce this on my iPad? That would allow me to see exactly why it is crashing and fix the issue quickly.
If that is not an option I would have some follow-up questions to help me narrow down the cause but I'd rather not take up any of your time with that if you can just send me the project.

Apologies for the inconvenience and thanks for the patience!

**Comment by keiwando** [comment]
This issue should now be fixed with the 1.7 update.

**Comment by falkthepolice** [comment]
And you added a perspective warp! Great stuff!On 2 Dec 2022, at 17:26, Keiwan Donyagard ***@***.***> wrote:﻿
This issue should now be fixed with the 1.7 update.
